gooey/cli.py

```diff
@@ -0,0 +1,35 @@
+'''The command line interface.'''
+
+import sys
+from importlib import import_module
+from importlib.metadata import entry_points
```

Comment:
We probably need to try and import it first, then `importlib_metadata` if it fails.